### PR TITLE
memorialize Anna Harren in the bastion of the turbofish

### PR DESCRIPTION
**Pull request by boringcactus**
this seems fitting, at least to me.

### CODE REVIEW

**Comment by steveklabnik** [comment]
r+ rollup

I wish I had something better to say here than "Thank you, Anna." But I do not. Thank you, Anna.

File: src/test/ui/bastion-of-the-turbofish.rs [hunk 1]
```diff
@@ -25,9 +25,12 @@
 //
 // My heart aches in sorrow, for I know I am defeated. Let this be a warning
 // to all those who come after. Here stands the bastion of the Turbofish.
+//
+// RIP Anna Harren, Guardian Angel of the Hallowed Turbofish. <3
 
 // See https://github.com/rust-lang/rust/pull/53562
 // and https://github.com/rust-lang/rfcs/pull/2527
+// and https://twitter.com/garblefart/status/1393236602856611843
```
Comment (glandium) [inline src/test/ui/bastion-of-the-turbofish.rs:33]: I wonder what's going to happen to this link. The Twitter policy for accounts of deceased people is, AIUI, not settled at the moment, but they were talking about expiring accounts at some point... Would it be better to put a web archive link?

Comment (estebank) [reply src/test/ui/bastion-of-the-turbofish.rs:33]: As long as the tweet itself isn't deleted, the link will survive account renaming, if it comes to that.

**Pull request merged**